sdk/eidolon_ai_sdk/util/schema_to_model.py

```diff
@@ -26,6 +26,36 @@
 }
 
 
+def flatten_refs(main_schema: Dict[str, Any], starting_node: Dict[str, Any]):
```

Comment:
is this different than jsonref.replace_refs ? If we can it would good to use the jsonref library so we don't need to maintain custom code

Reply:
I don't think it is.  I'll try to replace it with that an rerun tests.